theaters/calamity/theater/Kobuleti/Air Defense/SAM-Kobuleti-3-3.stm

```diff
@@ -51,7 +51,7 @@ staticTemplate =
                   {
                     coldAtStart = false,
                     heading = 0,
-                    name = "SAM-Kobuleti-3-3-1",
+                    name = "Primary Destroyed SAM-Kobuleti-3-3-1",
```

Comment:
Move "Primary Destroyed" to the end - having "SAM" at the beginning of the string is a convention used in the Air Intercept script to dynamically add/remove radars from the network